### DISCUSSION TITLE
Does it support cuda or only cpu?

### DISCUSSION THREAD

**Discussion by emelrad12** [NVIDIA-Omniverse/PhysX#14] [Q&A] [answered]
This page says "Get PhysX 5 CPU source on GitHub"
https://developer.nvidia.com/physx-sdk

So does that mean only features in the CPU column are available eg: no Soft Body Dynamics
Also none of them using cuda?

**Comment by amoravanszky** [comment, marked as answer]
Only CPU source code is included but CUDA based libraries / binaries are included for all the GPU features.
